modules/nixos/main.py

```diff
@@ -805,6 +533,12 @@ def run():
 
     # Write the configuration.nix file
     libcalamares.utils.host_env_process_output(["cp", "/dev/stdin", config], None, cfg)
+    # Format configuration file
+    libcalamares.utils.host_env_process_output(["nixfmt",  config], None)
+
+    # Copy modules
+    copyModules(root_mount_point, 'defaultConfigs/modules')
```

Comment:
Right now this leaves an empty `modules` folder in a users `/etc/nixos` dir. Maybe we should remove [modules/nixos/defaultConfigs/modules](https://github.com/NixOS/calamares-nixos-extensions/tree/39f4fdc00c111bf506c43d1025b914dbd43192da/modules/nixos/defaultConfigs/modules) directory and add a check to `copyModules` to return if the directory doesn't exist?

Reply:
I added the check, but have not tested the code again. The motivation was to either 
- add an minimal example module for new users there or,
- simply leaving the folder, with an .gitkeep file, for the future